ccd-definition/FixedLists/ExpertReportList.json

```diff
@@ -0,0 +1,163 @@
+    "DisplayOrder": 8
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "clinicalReportOnChild",
+    "ListElement": "Physiological Report on Child Only - Clinical",
+    "DisplayOrder": 9
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "educationalReportOnChild",
+    "ListElement": "Physiological Report on Child Only - Educational",
+    "DisplayOrder": 10
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "cognitiveReportOnParent",
+    "ListElement": "Physiological Report on Parent(s) - full cognitive",
+    "DisplayOrder": 11
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "functioningReportOnParent",
+    "ListElement": "Physiological Report on Parent(s) - functioning",
+    "DisplayOrder": 12
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "physiologicalReportOnParentAndChild",
+    "ListElement": "Physiological Report on Parent(s) and child",
+    "DisplayOrder": 13
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "multiDisciplinaryAssessment",
+    "ListElement": "Multi Disciplinary Assessment",
+    "DisplayOrder": 14
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "independentSocialWorker",
+    "ListElement": "Independent social worker",
+    "DisplayOrder": 15
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "haematologist",
+    "ListElement": "Haematologist",
+    "DisplayOrder": 16
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "opthamologist",
+    "ListElement": "Othamologist",
+    "DisplayOrder": 17
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "neurosurgeon",
+    "ListElement": "Neurosurgeon",
+    "DisplayOrder": 18
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "otherExpertReport",
+    "ListElement": "Other Expert Report",
+    "DisplayOrder": 19
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "professionalDrug",
+    "ListElement": "Professional: Drug/Alcohol",
+    "DisplayOrder": 20
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "professionalHair",
+    "ListElement": "Professional: Hair Strand",
+    "DisplayOrder": 21
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "professionalDNA",
+    "ListElement": "Professional: DNA testing",
+    "DisplayOrder": 22
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ExpertReportList",
+    "ListElementCode": "professionalOther",
+    "ListElement": "Professional: Other",
+    "DisplayOrder": 23
+  }
+]
```

Comment:
are you sure this definiton uploads? I seem to remember a 40 character limit on list elements in fixed lists 🤔

Reply:
uploads fine as we discussed. :)